cmd/geoipupdate/main.go

```diff
@@ -61,12 +61,12 @@ func run(client *http.Client, config *geoipupdate.Config) error {
 	for _, editionID := range config.EditionIDs {
 		filename, err := geoipupdate.GetFilename(config, editionID, client)
 		if err != nil {
-			return errors.Wrap(err, "error retrieving filename")
+			return errors.Wrapf(err, "error retrieving filename for %s", editionID)
 		}
 		filePath := filepath.Join(config.DatabaseDirectory, filename)
 		dbWriter, err := database.NewLocalFileDatabaseWriter(filePath, config.LockFile, config.Verbose)
 		if err != nil {
-			return errors.Wrap(err, "error creating database writer")
+			return errors.Wrapf(err, "error creating database writer for %s", editionID)
 		}
 		if err := dbReader.Get(dbWriter, editionID); err != nil {
 			return err
```

Comment:
Could we add it to this error as well? I think it'll catch more cases that will not have the edition currently.

Reply:
Sure, I'll add a `WithMessage`.